minai_plugin/sex.php

```diff
@@ -7,6 +7,7 @@
         $GLOBALS["ENABLED_FUNCTIONS"][]="ExtCmdStartVaginal";
         $GLOBALS["ENABLED_FUNCTIONS"][]="ExtCmdStartAnal";
         $GLOBALS["ENABLED_FUNCTIONS"][]="ExtCmdStartBlowjob";
+		$GLOBALS["ENABLED_FUNCTIONS"][]="ExtCmdStartHandjob";
```

Comment:
We have definitions for the hand job here in the php, but nothing in the command dispatcher in the skyrim plugin to process them.